msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+ms.author: v-sdhakshina
+ms.topic: Conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Preapproval of resource-specific consent (RSC) permissions allows admins to approve RSC based on individual app in Microsoft Teams. Initially, admins had the ability to turn on or turn off RSC permission in the Teams Admin Center to either approve or block all RSC enabled apps.
```

Comment:
"an individual app" or "individual apps" -- in Microsoft Teams

Reply:
Updated as suggested